docs/containers.conf.5.md

```diff
@@ -217,8 +217,8 @@ plugins.
 **network_config_dir**="/etc/cni/net.d/"
   Path to the directory where CNI configuration files are located.
 
-## LIBPOD TABLE
-The `libpod` table contains configuration options used to set up a libpod runtime.
+## ENGINE TABLE
+The `engine` table contains configuration options used to set up a container engines.
```

Comment:
maybe add something like "container engines such as Podman and Buildah."

Reply:
Fixed